### DISCUSSION TITLE
Are the order and approximate interval between visits within the same year and season preserved?

### DISCUSSION THREAD

**Discussion by MattHodgman** [MIT-LCP/mimic-code#1387] [MIMIC-III]
The documentation/publication states that the season of, and interval between, visits were preserved. Does this mean that two visits within the same season/month have the same order and approximate difference in days between them as before the shift? Is that information preserved enough to make patient predictions on a monthly basis?

**Comment by alistairewj** [comment]
The interval is preserved in MIMIC-IV but not the season. So two admissions for the *same* patient 30 days apart are likely to be in the same season - but they could be in different seasons if say the first admission was on the last day of summer.

In MIMIC-IV, you can't group across patients by season as it is randomized. In MIMIC-III you can.
